open-craft/xblock-eoc-journal
# Show progress and engagement in EOC journal

This PR adds user progress and engagement metrics in a course.

**Testing**

- Setup course with xblock-eoc-journal
- Make comments and posts in the discussion forum
- Complete tasks in the course
- Go to the EOC journal, check that the user and cohort progress display matches the one shown in the course home
- Checkt that the engagement metrics are displayed in a table.

<img width="1056" alt="screen shot 2017-07-10 at 20 32 48" src="https://user-images.githubusercontent.com/831084/28044660-1001dd04-65af-11e7-8bca-61885e1d63cf.png">

**Reviewers**
